filebeat/beater/filebeat.go

```diff
@@ -20,7 +20,7 @@ import (
 	"github.com/elastic/beats/filebeat/spooler"
 
 	// Add filebeat level processors
-	_ "github.com/elastic/beats/filebeat/processors/kubernetes"
+	_ "github.com/elastic/beats/filebeat/processor/kubernetes"
```

Comment:
@exekias I disagree on this one as we should change it for some time in libbeat to `processor`. It is `processors` for historical reasons.